survey-runner.tf

```diff
@@ -296,6 +296,10 @@ output "survey_launcher_for_ecs" {
   value = "${module.survey-launcher-for-ecs.service_address}"
 }
 
+output "survey_validator_for_ecs" {
```

Comment:
This doesn't need to be an output, it needs to be added here. https://github.com/ONSdigital/eq-terraform/pull/115/files#diff-964611b37b257ebaffb6708a708955f4R175